doc/Command-Reference.md

```diff
@@ -4223,6 +4223,7 @@ The type of interfaces include the following.
   admin@sonic:~$ show ip interfaces
   Interface       Master          IPv4 address/mask     Admin/Oper      BGP Neighbor     Neighbor IP     Flags
   -------------   ------------    ------------------    --------------  -------------    -------------   -------
+  Ethernet8.10                    1.1.1.1/24            down/down       N/A              N/A
```

Comment:
How is sub port interface oper status defined here? SAI does not have this attribute for sub port interface.

Reply:
'show ip interface' get the admin & oper status reported by the kernel.
(according to the code: sonic-utilities/show/main.py: get_if_admin_state and get_if_oper_state)